src/main/java/stsjorbsmod/powers/OverkillPower.java

```diff
@@ -0,0 +1,48 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+
+
+public class OverkillPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(OverkillPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    public OverkillPower(AbstractCreature owner, int amount) {
+        super(STATIC);
+        this.owner = owner;
+        this.type = PowerType.BUFF;
+        this.amount = amount;
+        updateDescription();
+    }
+
+    @Override
+    public void updateDescription() {
+        if(amount == 0) {
+            this.description = String.format(DESCRIPTIONS[0]);
+        }
+        else {
+            this.description = String.format(DESCRIPTIONS[1], this.amount);
+        }
+    }
+
+
+    @Override
+    public void onAttack(DamageInfo info, int damageAmount, AbstractCreature target) {
```

Comment:
Using this hook makes me a little nervous because in theory a monster power with an `onAttacked` hook could change the damage amount between this hook and the monster's death, but in practice I don't see any non-deprecated powers that use `onAttacked` to change damage amount, so I think this is probably fine